std/stdio.d

```diff
@@ -3872,7 +3878,7 @@ struct lines
             return opApplyRaw(dg);
         }
     }
-    // no UTF checking
+    /// Implements `opApply` `foreach` support but with no UTF checking
     int opApplyRaw(D)(scope D dg)
```

Comment:
This looks like it's just intended to be used by `opApply` and should not be treated as public.